### PR TITLE
relayer: Fix typescript to make PR 2391 Pass

### PR DESCRIPTION
**Pull request by derpy-duck**
…layer interfaces

### CODE REVIEW

File: ethereum/truffle-config.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const HDWalletProvider = require("@truffle/hdwallet-provider");
 const KLAYHDWalletProvider = require("truffle-hdwallet-provider-klaytn");
 
 module.exports = {
-  contracts_directory: "contracts/{*.sol,bridge/{*.sol,interfaces/*.sol,token/*.sol,mock/*.sol,utils/*.sol},interfaces/*.sol,mock/*.sol,nft/{*.sol,interfaces/*.sol,token/*.sol,mock/*.sol}}",
+  contracts_directory: "contracts/{bridge/{*.sol,interfaces/.sol,token/*.sol,mock/*.sol,utils/*.sol},interfaces/IWormhole.sol,mock/*.sol,nft/{*.sol,interfaces/*.sol,token/*.sol,mock/*.sol}}",
```
Comment (scnale) [inline ethereum/truffle-config.js:6]: Beware that the wormhole core contracts are stored directly in `contracts`.

Comment (derpy-duck) [reply ethereum/truffle-config.js:6]: Ah yes hadn't intended to remove that, thanks!

File: sdk/js/src/relayer/main/main.ts [hunk 2]
```diff
@@ -165,25 +165,23 @@ export async function send(
...
     await sourceCoreRelayer.getDefaultRelayProvider();
-  const sendStruct: SendStruct = {
-    targetChainId: targetChainId,
-    targetAddress: "0x" + tryNativeToHexString(targetAddress, "ethereum"),
-    refundChainId:
-      (refundLocationExists && sendOptionalParams.refundChainId) || sourceChainId,
-    refundAddress:
-      "0x" +
+
+  const tx = sourceCoreRelayer["send(uint16,bytes32,uint16,bytes32,uint256,uint256,bytes,(uint8,uint16,bytes32,uint64,bytes32)[],uint8,address,bytes)"](
```
Comment (scnale) [inline sdk/js/src/relayer/main/main.ts:169]: Perhaps a comment here could explain which `send` interface this is?

File: ethereum/truffle-config.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ const HDWalletProvider = require("@truffle/hdwallet-provider");
 const KLAYHDWalletProvider = require("truffle-hdwallet-provider-klaytn");
 
 module.exports = {
-  contracts_directory: "contracts/{*.sol,bridge/{*.sol,interfaces/*.sol,token/*.sol,mock/*.sol,utils/*.sol},interfaces/*.sol,mock/*.sol,nft/{*.sol,interfaces/*.sol,token/*.sol,mock/*.sol}}",
+  contracts_directory: "contracts/{*.sol,bridge/{*.sol,interfaces/.sol,token/*.sol,mock/*.sol,utils/*.sol},interfaces/IWormhole.sol,mock/*.sol,nft/{*.sol,interfaces/*.sol,token/*.sol,mock/*.sol}}",
```
[the remark this answers is not in the document]
Comment (derpy-duck) [reply ethereum/truffle-config.js:6]: yup oops

**Pull request merged**